cmd/destroy/cmddestroy.go

```diff
@@ -68,7 +68,7 @@ func (r *DestroyRunner) RunE(cmd *cobra.Command, args []string) error {
 
 	// Run the destroyer. It will return a channel where we can receive updates
 	// to keep track of progress and any issues.
-	ch := r.Destroyer.Run(inv)
+	ch := r.Destroyer.Run(inventory.WrapInventoryObj(inv))
```

Comment:
I do not believe this will work. As described elsewhere, this `InventoryFactoryFunc` will need to be set in the `Provider`.

Reply:
Updated the code to use the function from `provider.InventoryClient`.